traffic_stats/influxdb_tools/create/create_ts_databases.go

```diff
@@ -20,45 +20,81 @@ under the License.
 package main
 
 import (
-	"flag"
 	"fmt"
-	"os"
 
 	influx "github.com/influxdata/influxdb/client/v2"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
```

Comment:
The style makes sense, I was just trying to take the first pass at "Go-ifying" them. `pkg/errors` did get presented by Dave Cheney at the last GopherCon and I have to say that I have used it extensively, and it almost feels like an extension of the standard library.  I do agree that we can forego `cli` until absolutely necessary.